lib/events/auditlog.go

```diff
@@ -1000,7 +1000,7 @@ func (l *AuditLog) StreamSessionEvents(ctx context.Context, sessionID session.ID
 }
 
 // getLocalLog returns the local (file based) audit log.
```

Comment:
Is it guaranteed to by a file based audit log still?

Reply:
do you mean we should update comment that it "returns the local ExternalAuditLogger" or something else?

Reply:
Right